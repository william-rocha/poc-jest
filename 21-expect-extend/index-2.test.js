const { toMatchSnapshot } = require("jest-snapshot");

expect.extend({
  toMatchTrimmedSnapshot(received, length) {
    return toMatchSnapshot.call(
      this,
      received.substring(0, length),
      "toMatchTrimmedSnapshot"
    );
  }
});

it("stores only 10 characters", () => {
  expect("extra long string oh my gerd").toMatchTrimmedSnapshot(10);
});
