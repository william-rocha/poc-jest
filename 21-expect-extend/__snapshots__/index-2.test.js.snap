// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`stores only 10 characters: toMatchTrimmedSnapshot 1`] = `"extra long"`;
